pkgs/by-name/ac/ace-of-penguins/package.nix

```
@@ -38,7 +46,7 @@ stdenv.mkDerivation rec {
   desktopItems =
     let
       generateItem = gameName: {
-        name = "${pname}-${gameName}";
+        name = "${finalAttrs.pname}-${gameName}";
```

```suggestion
        name = "ace-of-penguins-${gameName}";
```
It's usually better to hardcode pname unless you're doing some generic package builder